Components/Utils/ModelRenderer.cs

```diff
@@ -78,7 +78,7 @@ public HtmlString ServerSetup()
             {
                 CdnEnv = _model.CdtsEnvironment.CDN,
                 Mode = Mode.SERVER,
-                Base = null,
+                Base = _model.Settings.SRIEnabled ? null : new SetupBase { SRIEnabled = null}, //base left null for defaults is SRI is true
```

Comment:
SRIEnabled should be set to false when Settings.SRIEnabled

(and also, the RenderSplashStuff   (LeavingSecureSite and the fact tthe SRIEnabled is false)

Reply:
done